docs/devguide/contributing.asciidoc

```diff
@@ -40,6 +40,18 @@ you'd like to share with others, open a PR to add it to our list of
 https://github.com/elastic/beats/blob/master/libbeat/docs/communitybeats.asciidoc[community
 Beats].
 
+
+[float]
+[[adding-new-dashboard]]
+=== Adding a New Dashboard
+
+If you want to create a new Dashboard, please read <<community-dashboards>>. You don't need to
```

Comment:
Don't capitalize "Dashboard" here. (We capitalize Beat because it is a product name.)